### PR TITLE
Update readme.md to include running init with npx

### PR DESCRIPTION
**Pull request by martypdx**
Added instructions for running `ava --init` with `npx` when installing ava locally and at least `npm@0.5.2`

### CODE REVIEW

**Comment by novemberborn** [comment]
Thanks for the PR @martypdx. Do you think we Should generally advocate for `npx ava`? Both for `--init` and just plain running tests? And remove the global install advise? @sindresorhus?

**Comment by martypdx** [comment]
My .02:

Yes, for `--init`, no for running tests. 

For required project dev dependency I would advocate purely installing locally. I only `npm -g` for utilities and tools that are _optionally_ used in a project.

Since `--init` is only run once, it makes sense to use `npx ava --init`.

For ongoing scripts (test running) I would advocate creating `script` `test` because it's needed by CI and as cli flags getting added beyond `ava` it provides a documented, standard way you know how to test the project.

**Comment by novemberborn** [comment]
Right. I think if we promote `npx ava --init` then it wouldn't make sense to install globally.

I agree that normally you should run tests through `npm t`, but often when developing I reach for `npx ava`. 

I think that means we no longer have to advocate installing AVA globally.

**Comment by martypdx** [comment]
@novemberborn updated the PR to just provide local install instructions. I don't use `yarn`, so I think from research that is right way to run `--init`, but haven't tried it myself

File: readme.md [hunk 1]
```diff
@@ -101,23 +100,8 @@ Your `package.json` will then look like this:
...
-```
-
-Alternatively using Yarn:
-
-```console
-$ yarn add --dev ava@next
-```
-
-You'll have to configure the `test` script in your `package.json` to use `ava` (see above).
+NOTE: Running `npx` requires `npm` version `0.5.2` or greater. Otherwise, you'll have to
```
Comment (novemberborn) [inline readme.md]: It's `5.2.0`, not `0.5.2` 😉 Perhaps add a link to https://github.com/npm/npm/releases/tag/v5.2.0?

**Comment by sindresorhus** [comment]
👍

**Comment by martypdx** [comment]
Updated to correct npm version with link to that update that outlines `npx`. Also made references to `package.json` and commands consistently code formatted

**Comment by novemberborn** [comment]
Thanks @martypdx!

**Pull request merged**